playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -230,6 +230,14 @@
     - reconcile_scc_result.rc == 0
     run_once: true
 
+  - name: Migrate storage post policy reconcile
```

Comment:
We should revisit why this entire play's tasks are run_once, I guess that's so that we can set the fact on all masters but only run the data manipulation stuff once.